ime/app/src/main/res/values-de/strings.xml

```diff
@@ -392,6 +392,7 @@
     <string name="extension_kbd_bottom_alt_with_mic">Alternativ mit Mikrofon</string>
     <string name="extension_kbd_bottom_old">Old School</string>
     <string name="extension_kbd_bottom_simple">Einfach</string>
+    <string name="extension_kbd_bottom_simple_with_emoji">Einfach mit Emojis</string>
```

Comment:
this should not be done here but at https://crowdin.net

Reply:
I have little experience with i18n in Android apps and was just following what was already here :)
Should I just omit this line or what are you suggesting?